Makefile

```
@@ -18,13 +18,20 @@ VERSION ?= $(shell git describe --match 'v[0-9]*' --dirty='.m' --always --tags)
 GO_LDFLAGS ?= -s -w -X ${PKG}/internal.Version=${VERSION}
 GO_BUILDTAGS ?= e2e,kube
 
-UNAME_S := $(shell uname -s)
-ifeq ($(UNAME_S),Linux)
+ifeq ($(OS),Windows_NT)
+    DETECTED_OS = Windows
+else
+    DETECTED_OS = $(shell uname)
```

I think you want `uname -s` here so that it will give `Linux`
```suggestion
    DETECTED_OS = $(shell uname -s)
```

---

Ahhh I didn't realize I lost that moving things around, I'll add it back! (Although I think by default `uname` will return the same thing as `uname -s` -- or so `man uname` says on Alpine and Ubuntu)